ledger-tool/src/blockstore.rs

```diff
@@ -600,81 +597,77 @@ pub fn blockstore_subcommands<'a, 'b>(hidden: bool) -> Vec<App<'a, 'b>> {
 }
 
 pub fn blockstore_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) {
+    do_blockstore_process_command(ledger_path, matches).unwrap_or_else(|err| {
+        eprintln!("Failed to complete command: {err:?}");
+        std::process::exit(1);
+    });
+}
```

Comment:
I think it'd be nicer if the subcommand functions all return a common `Result` type, and we can have a single `unwrap_or_else()` within `main.rs`.

However, I started working on updating the other commands, and hit some minor "snags" with `bigtable.rs`. Namely, `solana_ledger::bigtable_upload::upload_confirmed_blocks()` and `solana_ledger::bigtable_delete::delete_confirmed_blocks()` return a `Result<(), Box<dyn std::error::Error>`.

This was seemingly done as there are subcalls that can return a `BlockstoreError` or a `solana_storage_bigtable::Error`. Guessing the same dilemma is how it came to be that the bigtable ledger-tool commands return a `Result<(), Box<dyn std::error::Error>` as well.

Kind of the fence here - part of me wants to clean those up right now, but part of me is also thinking we have a decent logical boundary and could handle additional condensing in a followup. You leaning one way or the other ?

Reply:
Yeah, I started looking at the `bigtable` subcommand as well. While I haven't really dug in yet, I like the idea of cleaning those up lower and being able to use the LedgerToolError there as well.

But I see no real drawback in merging this one as-is, and doing the additional condensing later.